docs/tutorial/deployments.md

```diff
@@ -150,11 +150,20 @@ You should spend some time experimenting with this setup; a few next steps for e
 - use the UI to submit ad-hoc runs for the "sleeper" deployment with different values for `sleep`
 - use the UI to cancel an active run for the "sleeper" deployment (good luck cancelling the "fast" one 😉)
 
+!!! tip "Security Note"
```

Comment:
The SE team originally included this in the tutorial since its a major value prop. I moved it from the work pool page to the deployment page since its the case regardless of whether you're using serve or workers.